### PR TITLE
Fix Recently Modified filter to include closed issues

### PR DESCRIPTION
**Pull request by cproensa**
Fix "Hide status" property for the filter shown in my-view-page for
"Recently Modified" box. It should display closed issues.

### CODE REVIEW

File: my_view_inc.php [hunk 1]
```diff
@@ -75,7 +75,9 @@
 
 # @TODO cproensa: make this value configurable
 $t_recent_days = 30;
-$c_filter['recent_mod'] = filter_create_recently_modified( $t_recent_days );
+$t_recent_mod_filter = filter_create_recently_modified( $t_recent_days );
+$t_recent_mod_filter[FILTER_PROPERTY_HIDE_STATUS] = META_FILTER_NONE;
```
Comment (vboctor) [inline my_view_inc.php]: Why wouldn't this be part of filter_create_recently_modified().  This is the only place that we use this API and should produce the proper filter that can be re-used elsewhere overtime, like in the SOAP API or other places.

**Comment by vboctor** [comment]
@cproensa please also change the PR to target `master-2.1` since it is a regression and a simple fix.

File: my_view_inc.php (hunk 1 above)
Comment (cproensa) [inline my_view_inc.php]: filter_create_recently_modified(), when used to create a new filter, sets the date filter to the needed days values, and keeps the rest of properties as given by a new default filter.
Hide status is then returned as the default value as per configuration "hide_status_default".

We seem to support that this filter box must show all status, regardless of "hide_status_default" configuration, so this property makes sense to be modified in my-view-page for this specific usage.

**Comment by cproensa** [comment]
Rewritten with victor suggestion

**Comment by vboctor** [comment]
@cproensa please update the comment with reference to the issue in MantisBT bug tracker.

**Comment by vboctor** [comment]
@cproensa also please target `master-2.1`.

**Comment by cproensa** [comment]
>  update the comment with reference
> also please target master-2.1.

all sorted out now

**Pull request merged**